modules/storages/app/common/storages/peripherals/storage_interaction/one_drive/set_permissions_command.rb

```diff
@@ -45,25 +45,33 @@ def delete? = permission_ids.any? && user_ids.empty?
             def update? = permission_ids.any? && user_ids.any?
           end
 
-          def self.call(storage:, auth_strategy:, path:, permissions:)
-            new(storage).call(auth_strategy:, path:, permissions:)
+          def self.call(storage:, auth_strategy:, input_data:)
```

Comment:
We probably should add some YARD docs here on what is the `input_data`, so it makes easier for the IDEs/Editors/Our future selves

Reply:
Good idea.